genome/gene_clust.py

```
@@ -1,126 +1,126 @@
-        u100 = pd.read_csv(self.u100, sep="\t", header=None, names=["U100", "All"])
-        u90 = pd.read_csv(self.u90, sep="\t", header=None, names=["U90", "U100"])
-        u50 = pd.read_csv(self.u50, sep="\t", header=None, names=["U50", "U90"])
-        rep2all = u50.merge(u90).merge(u100)[keep_]
+    TSV_COLS: tuple[str, ...] = ("All",)
+
+    def _load_rep2all(self, *files):
+        rep2all: pd.DataFrame
+        for i, f in enumerate(files):
+            names = self.TSV_COLS[i : i + 2][::-1]
+            df = pd.read_csv(f, sep="\t", header=None, names=names)
+            rep2all = df if i == 0 else rep2all.merge(df)
```

Ruff (0.11.9)

Undefined name `rep2all`

(F821)